src/shim/state_tree.rs

```diff
@@ -94,6 +94,7 @@ impl TryFrom<StateTreeVersion> for StateTreeVersionV3 {
 /// Not all the inner methods are implemented, only those that are needed. Feel
 /// free to add those when necessary.
 pub enum StateTree<S> {
+    V0(state_tree_v0::StateTreeV0<Arc<S>>),
```

Comment:
Could you please elaborate on why it's needed?

Reply:
Done. Wrote which versions are supported by FVM2 and FVM3.